src/org/jitsi/impl/neomedia/transform/dtls/DtlsPacketTransformer.java

```diff
@@ -702,20 +702,23 @@ private SinglePacketTransformer initializeSRTPTransformer(
                     auth_key_length,
                     RTP_auth_tag_length,
                     cipher_salt_length);
+        org.jitsi.utils.logging2.Logger logger2 = new org.jitsi.utils.logging2.LoggerImpl(getClass().getName());
```

Comment:
does this same file use both logging2 and logging?

Reply:
Yes.  I didn't want to start pulling the ball of yarn that would be switching all of libjitsi over to logging2.

Reply:
Why. WHY? Why aren't we just switching to sl4fj!?
Ditch our own shit! This is all from the time without proper libraries around.

Reply:
This is @bbaldino's new logging framework so I'll tag him in here.

Reply:
@ibauersachs i don't disagree, but that's a much bigger change than i wanted to get into right now.

Reply:
I can't follow. What would be the big change?
In logging(1), the `Category` can be mapped to markers, the levels would be some `sed -i` commands. In logging2 the probably not-so-easy thing is the context, but that could be implemented around slf4j too.